### ISSUE TITLE
run onnx raise error? is Constant node  discarsed?

### ISSUE THREAD

**Issue by cqray1990** [onnx/onnx#4345]
System information

    OS Platform and Distribution (e.g. Linux Ubuntu 16.04): 16.04
    ONNX version (e.g. 1.7): 1.8
    Python version:3.6.8
    GCC/Compiler version (if compiling from source):
    CMake version:
    Protobuf version:
    Visual Studio version (if applicable):

export onnx:
dynamic_axes = {"input": {2: "height", 3: "width"},"output": {2: "height", 3: "width"}}
torch.onnx.export(model.model,
x,
onnx_file_name,
export_params=True,
verbose=True,
opset_version=11,
do_constant_folding=True,
input_names=input_names, output_names=output_names,
dynamic_axes=dynamic_axes)
i check onnxmodel no errors:
import onnx
model = onnx.load("./mm_dy.onnx")
onnx.checker.check_model(model)
onnx.helper.printable_graph(model.graph)

but when i run onnx model raise error:
onnxruntime.capi.onnxruntime_pybind11_state.RuntimeException: [ONNXRuntimeError] : 6 : RUNTIME_EXCEPTION : Non-zero status code returned while running Add node. Name:'' Status Message: /onnxruntime_src/onnxruntime/core/providers/cpu/math/element_wise_ops.h:479 void onnxruntime::BroadcastIterator::Init(int64_t, int64_t) axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 40 by 44

finally i compare my code and find if use follows code,the error raise

def _upsample_add(self, x, y):
    _, _, H, W = map(int, y.size()) ----->will be converted to Constant Node 
    return F.interpolate(x, size=(H,W),mode="nearest") + y

but when use follows code: there is no errors , why????

def _upsample_add(self, x, y):
    return F.interpolate(x, size=y.size()[2:]) + y

**Comment by jcwchen** [comment]
It seems exactly the same as https://github.com/onnx/onnx/issues/4339. Please track this issue there and I will close this one. Feel free to reopen it if you still have other ONNX questions.
